client/test/cypress/support/index.js

```diff
@@ -20,9 +20,10 @@ import "./commands"
 const resizeObserverLoopError = "ResizeObserver loop limit exceeded"
 
 Cypress.on("uncaught:exception", (err) => {
-  if (err.message.includes(resizeObserverLoopError)) {
-    // returning false here prevents Cypress from
-    // failing the test
-    return false
-  }
+  // if (err.message.includes(resizeObserverLoopError)) {
+  //   // returning false here prevents Cypress from
+  //   // failing the test
+  //   return false
+  // }
+  return false
```

Comment:
```suggestion
   if (err.message.includes(resizeObserverLoopError)) {
     // returning false here prevents Cypress from
     // failing the test
     return false
   }
  return false
```
Not sure if this was commented out during debugging and should be put back in or if its really done and should just get deleted.  (I've little pushing me in either direction here)

Reply:
Yeah, this was commented out for debugging and I forgot to revert. I'll revert now.